corehq/motech/repeaters/static/repeaters/js/bootstrap3/repeat_record_report.js

```diff
@@ -395,15 +394,13 @@ hqDefine('repeaters/js/bootstrap3/repeat_record_report', [
 
         function checkMultipleItems(action) {
             for (const item of items) {
-                const id = item.getAttribute('data-id');
-                const query = `[data-record-id="${id}"][class="btn btn-default ${action}-record-payload"]`;
-                const button = document.querySelector(query);
-                if (!!button && item.type === 'checkbox') {
-                    if (item.checked) {
-                        item.checked = false;
-                    } else {
-                        item.checked = true;
-                    }
+                const isQueued = item.getAttribute('is_queued');
```

Comment:
You might call `parseInt(item.getAttribute('is_queued'))` here, or even `!!parseInt(item.getAttribute('is_queued'))` so you're just dealing with booleans in the rest of the code.

Reply:
I realized I was making the logic more complicated than it needed to be, and also doing it differently in two different places, so this commit resolved those issues as well as used the more boolean conversion logic: ea7eaf4463e40497b32e46e13256d2048bc4c4cc